tap_sftp/client.py

```
@@ -74,7 +74,7 @@ def __connect(self):
                 LOGGER.info('Connection failed, retrying...')
                 if i >= (self.retries):
                     if isinstance(ex, AuthenticationException):
-                        raise SymonException('Authentication failed. Please check your credentials.','sftp.AuthenticationError')
+                        raise SymonException('Authentication failed. Please check your credentials and SFTP server availability.','sftp.AuthenticationError')
```

What's the difference between this case and the sftp.ServerError case below?
Doesn't seem like we can distinguish between availability/port/credential issue; maybe we just raise a generic exception in all cases?

---

If we try to connect to the sftp server with host/username/password all correct but the port is wrong, then we will actually run into the 2nd error instead